packages/react-menu/src/MenuNestedSubmenus.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { Menu, MenuTrigger, MenuList, MenuItem, MenuPopover } from './index';
+
+import { Button } from '@fluentui/react-button';
+
+const EditorLayoutSubMenu = () => {
```

Comment:
I do not see any submenu indicators. Is that expected?

Reply:
No it's not, there might have been some kind of regression for the icon, since we switched to the system icons. Will take a look at that